### DISCUSSION TITLE
PlugData on Raspberry Pi

### DISCUSSION THREAD

**Discussion by ghost** [plugdata-team/plugdata#131] [Q&A] [answered]
Dear plugdata developers, 
Is it possible to run plugdata standalone directly on raspbian without compiling it? I tried to compile it with cmake, but my raspberry pi 3B crashes trying to build juceaide. Otherwise, would you have any advice on how to successfully compile it?
Thanks a lot :)

**Comment by timothyschoen** [comment, marked as answer]
@KottV maintains Linux packages here:

https://software.opensuse.org//download.html?project=home%3Abespokesynth&package=plugdata

These will move to another link soon, but there is a raspbian build there!

**Reply by ghost** [reply]
Thank you very much for your reply. I'm trying to install it, then I'll tell you as soon as it works ^^

**Reply by ghost** [reply]
Hello, sorry for my late reply. I had tested with the arm64 compiled version, but I could not launch plugdata after the installation. With the armhf version, plugdata runs without any problem. Thanks for your help!

**Reply by KottV** [reply]
it's here now https://build.opensuse.org/package/show/home:plugdata/plugdata

**Comment by timothyschoen** [comment]
I'm curious what errors juceaide is giving you though?

**Reply by ghost** [reply]
In fact, the raspberry pi does not stop, but it is extremely slow and never finishes compiling (waited about one day :D )
